app/controllers/ApiController.scala

```diff
@@ -145,7 +147,7 @@ final class ApiController @Inject()(api: OreRestfulApi,
 
   private def error(key: String, error: String) = Json.obj("errors" -> Map(key -> List(this.messagesApi(error))))
 
-  def deployVersion(version: String, pluginId: String, name: String) = ProjectAction(pluginId) { implicit request =>
+  def deployVersion(version: String, pluginId: String, name: String) = (Action andThen projectAction(pluginId)) { implicit request =>
```

Comment:
Why was `ProjectAction` removed here?

Reply:
The behavior is not removed. projectAction does exactly the same, we had to change this because of a the action-refactor in 2.6.

Reply:
But [`OreBaseController` has `ProjectAction`](https://github.com/SpongePowered/Ore/pull/498/files#diff-022b1f4b21acedf492f737eb21692398R81) in it.